Mathlib/RingTheory/IntegralRestrict.lean

```diff
@@ -268,6 +268,26 @@ lemma Algebra.intTrace_eq_of_isLocalization
   rw [← IsScalarTower.algebraMap_apply, Algebra.algebraMap_intTrace_fractionRing,
     Algebra.algebraMap_intTrace (L := L), ← IsScalarTower.algebraMap_apply]
 
+variable {A} in
+lemma isIntegrallyClosed_of_isLocalization [IsIntegrallyClosed A] [IsDomain A] (M : Submonoid A)
```

Comment:
Can this go in `Mathlib.RingTheory.IntegrallyClosed` instead?

Reply:
No, since it needs [IsFractionRing.isFractionRing_of_isDomain_of_isLocalization](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/Localization/LocalizationLocalization.html#IsFractionRing.isFractionRing_of_isDomain_of_isLocalization). It cannot go to `Mathlib.RingTheory.Localization.LocalizationLocalization` either, since that file doesn't know what `IsIntegrallyClosed` means. I can Of course change a little the imports.

Reply:
Maybe we need a file for about licalization and integral closure.

Reply:
I think it makes sense to add `Mathlib.RingTheory.Localization.LocalizationLocalization` to the import of `Mathlib.RingTheory.IntegrallyClosed`.

Reply:
OK, that makes sense. Anyway we should probably split `IntegrallyClosed` into `defs` `basic` etc. (I will do in another PR.)